tests/dcft10/input.dat

```diff
@@ -0,0 +1,34 @@
+#! The multiple guesses for DCT amplitudes for ODC-12.
+
+refodc12     = -5.7749813965267096 #TEST
+
+molecule he2 {
+    He
+    He 1 3.2
+}
+
+set {
+    r_convergence 12
+    ao_basis    none
+    algorithm   simultaneous
+    basis       6-31G**
+    df_scf_guess false
+    reference   uhf
+}
+
+set dcft_functional odc-12
+energy('dcft')
+compare_values(refodc12, variable("DCFT TOTAL ENERGY"), 10, "ODC-12 Energy");                #TEST
+
+set dcft_guess cc
+energy('dcft')
+compare_values(refodc12, variable("DCFT TOTAL ENERGY"), 10, "ODC-12 Energy");                #TEST
+
+set dcft_guess bcc
+wfn = energy('dcft', return_wfn=True)[1]
+compare_values(refodc12, variable("DCFT TOTAL ENERGY"), 10, "ODC-12 Energy");                #TEST
```

Comment:
Is it possible / advisable to limit the number of iterations to just a couple here?  That way we'd be testing not only that the run converges to the correct solution, but that the starting guess is correctly read in.

Reply:
No; the `cc` and `bcc` starting guesses are _worse_ than the MP2 guess for this system, at about 5 times the RMS error in the amplitudes. Each takes 12 iterations.

When you read in the amplitudes and orbitals (`dcft_guess dcft` with `ref_wfn` passing), the guess is perfect, so I already set the number of iterations to 3 there.

Reply:
Ah, my bad!  I missed the line where you set the number of iterations to 3